pallets/crowdloans/src/lib.rs

```diff
@@ -579,6 +579,13 @@ pub mod pallet {
         pub fn slot_expired(origin: OriginFor<T>, crowdloan: ParaId) -> DispatchResult {
             T::SlotExpiredOrigin::ensure_origin(origin)?;
 
+            // inital read to check if vault exists and is in the correct phase
+            let vault = Self::current_vault(crowdloan).ok_or(Error::<T>::VaultDoesNotExist)?;
+            ensure!(
```

Comment:
this doesn't make sense because `expired` will need to come from `succeeded` now

Reply:
and we will need to have a call to migrate from `Closed` to `Succeeded`